docs/docs/features/requests-result.mdx

```diff
@@ -126,6 +126,21 @@ export function fetchTodos() {
 }
 ```
 
+- `cacheResponseData` - Cache the response data and emit it as data for skipped requests, defaults to `false`
+
+```ts title="todos.service.ts"
+import { trackRequestResult } from '@ngneat/elf-requests';
+import { setTodos } from './todos.repository';
+
+export function fetchTodos() {
```

Comment:
Subscribe to fetchTodos and show that we can access the data property

Reply:
done